cmd/minishift/cmd/start_preflight.go

```diff
@@ -584,9 +594,12 @@ func checkVMDriver() bool {
 }
 
 func validateOpenshiftVersion() bool {
-	requestedVersion := viper.GetString(configCmd.OpenshiftVersion.Name)
+	requestedOpenShiftVersion, err := getOpenShiftVersion()
+	if err != nil {
+		atexit.ExitWithMessage(1, fmt.Sprintf("Error getting OpenShift version: %v", err))
+	}
```

Comment:
why don't change the function signature so it will take version also, now you don't need to explicitly call the `getOpenshiftVersion()` multiple places and handle the error?

Reply:
@praveenkumar What is the expected signature here?